datasets/klue/klue.py

```diff
@@ -0,0 +1,532 @@
+                            sentence = row[1]
+                        else:
+                            char.append(row[0])
+                            ne_tag.append(row[1])
+                    else:  # new line
+                        assert len(char) == len(ne_tag)
+                        yield id_, {"sentence": sentence, "char": char, "ne_tag": ne_tag}
+
+        if self.config.name == "dp":
+            with open(data_file, encoding="UTF-8") as f:
+                reader = csv.reader(f, delimiter="\t", quoting=csv.QUOTE_NONE)
+                for _ in range(5):  # skip headers
+                    next(reader)
+                id_ = -1
+                for row in reader:
+                    if row:
+                        if row[0].startswith("##"):
+                            id_ += 1
+                            index = []
+                            word_form = []
+                            lemma = []
+                            pos = []
+                            head = []
+                            deprel = []
+                            sentence = row[1]
+                        else:
+                            index.append(row[0])
+                            word_form.append(row[1])
+                            lemma.append(row[2])
+                            pos.append(row[3])
+                            head.append(row[4])
+                            deprel.append(row[5])
+                    else:  # new line
+                        assert len(index) == len(word_form) == len(lemma) == len(pos) == len(head) == len(deprel)
+                        yield id_, {
+                            "sentence": sentence,
+                            "index": index,
+                            "word_form": word_form,
+                            "lemma": lemma,
+                            "pos": pos,
+                            "head": head,
+                            "deprel": deprel,
+                        }
+
+        if self.config.name == "mrc":
+            with open(data_file, encoding="UTF-8") as f:
+                f = json.load(f)
+                for example in f["data"]:
+                    title = example.get("title", "")
+                    news_category = example.get("news_category", "")
+                    source = example["source"]
+                    for id_, paragraph in enumerate(example["paragraphs"]):
+                        context = paragraph["context"].strip()
+                        for qa in paragraph["qas"]:
+                            guid = qa["guid"]
+                            question_type = qa["question_type"]
+                            is_impossible = qa["is_impossible"]
+                            question = qa["question"].strip()
+
+                            answer_starts = [answer["answer_start"] for answer in qa["answers"]]
+                            answers = [answer["text"].strip() for answer in qa["answers"]]
```

Comment:
Some examples only have not `"answers"` but `"plausible_answers"`.  Therefore, you should fix as follows.
```suggestion
                        if qa["answers"]:
                            answer_starts = [answer["answer_start"] for answer in qa["answers"]]
                            answers = [answer["text"].strip() for answer in qa["answers"]]
```

see an example below.

```json
      {
          "title": "도전적인 일, 리더에 대한 신뢰, 협력 문화 …...가고 싶다, 이 회사 … 베인&컴퍼니·트위터",
          "paragraphs": [
              {
                  "context": "가장 일하고 싶은 회사로 미국 컨설팅 업체인 베인&컴퍼니가 선정됐다. 트위터와 링크트인, 페이스북 등 정보기술(IT) 기업에 대한 선호도 높게 나타났다. 미국 경제전문지 포브스는 “채용 사이트인 글라스도어가 미국에서 일하는 50만명의 직장인을 대상으로 설문조사를 벌여 ‘2014년 일하고 싶은 50대 회사’를 선정했다”고 11일(현지시간) 보도했다. 베인&컴퍼니는 2012년에 이어 두 번째로 1위에 올랐다. 하는 일이 재미있고 일이 영향력이 있으며, 함께 일하는 사람들이 똑똑해서 많이 배울 수 있다는 것 등이 직원들이 베인&컴퍼니에 만족하는 주된 이유였다. 서맨사 주판 글라스도어 대변인은 “베인&컴퍼니 직원들은 늘 새로운 것을 배울 수 있고 도전적인 일이 많아 하루도 지루한 날이 없다고 평가했다”며 “이런 경험은 회사를 그만둔 후에도 개인 경력에 도움이 된다”고 말했다. 트위터가 2위를 차지했다. 트위터 직원들은 복잡하고 중요한 일을 매일 하면서 전 세계 수백만명의 사용자들과 소통할 수 있다는 점을 회사에 다니고 싶은 이유로 꼽았다. 트위터에 이어 링크트인과 페이스북이 각각 3위, 5위를 차지하는 등 IT 기업의 선전도 눈에 띄었다. 글라스도어는 “톱 50 가운데 22개가 IT 기업”이라며 “서로 더 좋은 인재를 영입하고, 기존의 인재를 뺏기지 않기 위해 경쟁하는 것이 더 매력적인 기업환경을 만들기 위한 노력으로 이어지고 있다”고 설명했다. 2009년부터 일하고 싶은 회사를 선정해 발표하고 있는 글라스도어는 2012년 11월부터 지난달까지 회사에 대한 만족도, 임금이나 복지 혜택, 일과 가정 사이의 균형, 경영자에 대한 평가 등 18개의 질문으로 구성된 설문조사를 실시했다. 글라스도어는 “구직자들이 어느 회사에서 일할지 결정하는 데 좋은 정보를 주는 자료”라며 “특히 현장에서 일하는 직원의 평가라는 점에서 리스트에 오른 기업들은 자부심을 가져도 될 것”이라고 평가했다. 상위권에 오른 기업들은 공통점을 가지고 있었다. 미국 경제경영전문지 패스트컴퍼니는 일하고 싶은 기업의 6가지 특징으로 목표를 분명하게 설정하는 것, 일하는 과정에서 협동을 중요시하는 것, 도전적인 일을 하는 것, 회사가 꾸준히 발전하는 것, 리더가 자신감을 갖고 투명하게 경영하는 것, 충분한 보상을 하는 것 등을 꼽았다.",
                  "qas": [
                      {
                          "question": "퇴사하고 싶은 회사로 트위터가 순위에 선정된 해는?",
                          "answers": [],
                          "plausible_answers": [
                              {
                                  "text": "2014년",
                                  "answer_start": 155
                              },
                              {
                                  "text": "2014",
                                  "answer_start": 155
                              }
                          ],
                          "question_type": 3,
                          "is_impossible": true,
                          "guid": "klue-mrc-v1_dev_03566"
                      },
                      {
                          "question": "2014년 일하고 싶은 50대 회사 중에서 5위로 선정된 기업은?",
                          "answers": [
                              {
                                  "text": "페이스북",
                                  "answer_start": 543
                              }
                          ],
                          "question_type": 2,
                          "is_impossible": false,
                          "guid": "klue-mrc-v1_dev_02673"
                      },
                      {
                          "question": "포브스의 2014년 일하고 싶은 50대 회사 조사에서 5위를 한 기업은?",
                          "answers": [
                              {
                                  "text": "페이스북",
                                  "answer_start": 543
                              }
                          ],
                          "question_type": 1,
                          "is_impossible": false,
                          "guid": "klue-mrc-v1_dev_05664"
                      }
                  ]
              }
          ],
          "news_category": "국제",
          "source": "hankyung"
      }
```